### PR TITLE
Adapt pr-lint action to handle md names too

### PR DESCRIPTION
**Pull request by teoli2003**
In #7119, pr-lint didn't catch the useless pull request title as it was in the JS area, newly converted to Markdown.

This fixes this.

### CODE REVIEW

File: pr-lint/src/index.js [hunk 1]
```diff
@@ -17,15 +17,15 @@ async function main() {
...
+  if ((title === "Update index.html") || (title === "Update index.md")) {
+    // These are the default titles you get when you use the GitHub UI to
     // edit a file and following web form steps to create a PR.
     // These PRs are hard to triage and to organize because of the
     // rather "useless" title so let's put the onus back on the author
     // to come up with a better title.
     // See https://github.com/mdn/content/issues/782
     return (
-      'Pull request title can\'t just be "Update index.html"\n' +
+      'Pull request title can\'t just be "Update index.html" or "Update index.md"\n' +
```
Comment (peterbe) [inline pr-lint/src/index.js:28]:
```suggestion
      `Pull request title can't just be "${title}"\n` +
```

**Review by peterbe — CHANGES_REQUESTED** [review]
Great catch! But let's be smart about mentioning the `title` as part of the box message.

File: pr-lint/src/index.js [hunk 2]
```diff
@@ -17,15 +17,15 @@ async function main() {
   }
 
   const { title } = contextPullRequest;
-  if (title === "Update index.html") {
-    // This is the default title you get when you use the GitHub UI to
+  if ((title === "Update index.html") || (title === "Update index.md")) {
```
Comment (peterbe) [inline pr-lint/src/index.js:20]:
```suggestion
  if (title === "Update index.html" || title === "Update index.md") {
```

I'm not sure why `prettier` didn't complain about the excess brackets. Hmm..

**Review by peterbe — APPROVED** [review]
Just one little nit if you wouldn't mind merging that first, then I'll squash and merge.

**Comment by teoli2003** [comment]
Done.

**Pull request merged**